package.json

```diff
@@ -26,15 +26,15 @@
     },
     "extraResources": [
       {
-        "from": "dist/proxy",
-        "to": "proxy"
+        "from": "proxy/target/release/radicle-proxy",
```

Comment:
Not sure this will work, but we can try it out.

Reply:
I tried it and it works